src/sentry/models/file.py

```diff
@@ -58,7 +58,7 @@ def enum(**named_values):
 def _get_size_and_checksum(fileobj):
     size = 0
     checksum = sha1()
-    while 1:
+    while True:
```

Comment:
not that it matters much but `while 1` is optimized in python 2 and `while True` is not.

Reply:
`autopep8` changed those automatically, so not really sure we want to fight it.